src/go/k8s/tests/e2e/kafkaapi-client-auth/03-pandaproxy-produce.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: produce-message
+spec:
+  template:
+    spec:
+      containers:
+        - name: rpk
+          image: localhost/redpanda:dev
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          command:
+            - /bin/bash
+            - -c
+            - -ex
+          args:
+            - |
+              curl -s -v -X POST "http://cluster-tls-0.cluster-tls.$POD_NAMESPACE.svc.cluster.local:8082/topics/test" \
+              -H "Content-Type: application/vnd.kafka.json.v2+json" \
+              -d '{"records":[{"value":"Vectorized"},{"value":"Pandaproxy"},{"value":"JSON Demo"}]}' \
```

Comment:
I reused an existing test, I just renamed it. I also found out it does not fail, I thought I adjusted it though but perhaps did not 🤔 I'll fix that

Reply:
no I think my test works, I've added grep -v `error_code` which means that it fails if that string is added. I had to do it that way because the proxy returns HTTP 200 but an error code (at least in case of missing tls broker config)